lib/si-layer-cache/src/disk_cache.rs

```diff
@@ -0,0 +1,65 @@
+use std::path::Path;
+
+use sled::{self, IVec};
```

Comment:
Do we care that this is beta? Seems like the maintainer is low-key rewriting it prior to a 1.0 release

Reply:
for now I think ti's fine, only because we don't care about the data. we should move it to rocksdb eventually, but it's going to take some time to get a stable build of the rocksdb rust bindings